src/txn/txn_rollback_to_stable.c

```diff
@@ -8,6 +8,9 @@
 
 #include "wt_internal.h"
 
+/* Enable rollback to stable verbose messaging during recovery*/
+#define WT_VERB_RECOVERY_RTS_FLAG(session) \
```

Comment:
I think it would be better to provide a connection as an argument because we convert it in the macro anyways. So it could be:

```
#define WT_VERB_RECOVERY_RTS_FLAG(conn) \
(F_ISSET((conn), WT_CONN_RECOVERING) ? WT_VERB_RECOVERY | WT_VERB_RTS : WT_VERB_RTS)
```

Also, notice extra parenthesis around the argument "(conn)", that prevents against complicated statements caller might place in the argument.

Reply:
I will do the necessary changes. Thank you.